packages/snap-tracker/src/Tracker.ts

```diff
@@ -418,7 +424,7 @@ export class Tracker {
 			},
 		},
 		cart: {
-			view: (data: CartViewEvent, siteId?: string): BeaconEvent | undefined => {
+			view: (data: CartViewEvent, siteId?: string, currency?: string): BeaconEvent | undefined => {
```

Comment:
I think we can remove the currency param here and instead assume that the currency has already been set on the context.